test/spec/fixtures/schema.sql

```diff
@@ -2631,12 +2635,12 @@ create table plate_plan_step (
 	  REFERENCES well(well_id)
 );
 
-CREATE FUNCTION test.return_scalar_xml() RETURNS pg_catalog.xml
+CREATE FUNCTION test.return_scalar_xml() RETURNS "text/xml"
 LANGUAGE sql AS $$
   SELECT '<my-xml-tag/>'::pg_catalog.xml
 $$;
```

Comment:
This means https://postgrest.org/en/stable/how-tos/create-soap-endpoint.html will have to be changed to:

```sql
CREATE OR REPLACE FUNCTION my_soap_endpoint(xml) RETURNS "text/xml" AS $$
```

In theory we could assume `RETURNS xml` means `text/xml` but that would only work for scalars. It would be messy in the code.

The change is not so bad anyway, just needs a new line for `CREATE DOMAIN "text/xml" AS xml`